editor/project_converter_3_to_4.cpp

```diff
@@ -3031,9 +3030,9 @@ Vector<String> ProjectConverter3To4::check_for_rename_gdscript_functions(Vector<
 	return found_renames;
 }
 
-// TODO, this function should run only on all .gd files and also on lines in tscn files which
+// TODO, this function should run only on all ".gd" files and also on lines in ".tscn" files which
 void ProjectConverter3To4::process_gdscript_line(String &line, const RegExContainer &reg_container, bool builtin) {
-	// In this and other functions, reg.sub are used only after checking line with str.contains function which is sometimes few times faster with bigger line lengths
+	// In this and other functions, reg.sub() are used only after checking line with str.contains() function which is sometimes few times faster with bigger line lengths
```

Comment:
I would split this into multiple different sentences to make it easier to understand. Also, there’s subject-verb disagreement. I would change it to:

```cpp
	// In this and other functions, reg.sub() is used only after checking lines with str.contains().
	// When you have longer lines, doing so is sometimes significantly faster.
```